cmake/project.cmake.in

```diff
@@ -146,148 +146,6 @@ function(config_dir key)
 	return("${TMP_DIR}")
 endfunction()
 
```

Comment:
If the generator is also able to use the old way of generating a config, maybe we can add already so the find package for the generator here. Then all the plugins use the new stuff automatically...

Reply:
Agrees, I have done the changes required.